drp.py

```diff
@@ -51,7 +51,7 @@ def base_activity():
     'ruby',
     'html',
     'rust',
-    'v',
+    'v'
```

Comment:
I don't really think this is necessary, it's probably better to keep the comma there for consistency since it won't error because of it.

Reply:
Added it back